tests/hopp/test_dispatch.py

```diff
@@ -332,6 +339,100 @@ def create_test_objective_rule(m):
 
     # TODO: Update the simulate_with_dispatch function for towers and troughs
 
+def test_wave_dispatch():
+    expected_objective = 74403.6
+    dispatch_n_look_ahead = 48
+
+    data = {
+		"lat": 44.6899,
+		"lon": 124.1346,
+		"year": 2010,
+		"tz": -7,
+	}
+	
+    wave_resource_file = Path(__file__).absolute().parent.parent.parent / "resource_files" / "wave" / "Wave_resource_timeseries.csv"
+    site = SiteInfo(data, solar=False, wind=False, wave=True, wave_resource_file=wave_resource_file)
+
+    default_fin_config = {
```

Comment:
you can use `from tests.hopp.utils import DEFAULT_FIN_CONFIG` instead of copying this dict